spec/coffee/showIn.spec.coffee

```diff
@@ -11,6 +16,8 @@ describe 'Marionette.Component', ->
 
       # Stubs and spies
       @showViewSpy          = @sinon.spy(Component.prototype, '_showView')
+      @getViewSpy          = @sinon.spy(Component.prototype, '_getView')
+      @optionsViewSpy       = @sinon.spy(_, 'result')
```

Comment:
Instead of testing `_getView` and `_.result`, we should tested the expected outcome. I would like to see specs that test that the view ends up receiving the properties we supply via `viewOptions`.

Ideally, a spec would test that after showing in a region, the component's view's model is the one we passed in.

We should also test that our custom property gets passed along to the view and equals the value we expect. To confirm it does, we could use `@component.view.getOption('foo')` I believe.

Those tests should be repeated when options are a function as well.